test/pom.xml

```diff
@@ -245,6 +245,12 @@ THE SOFTWARE.
           </dependency>
         </dependencies>
       </plugin>
+      <plugin>
+        <artifactId>maven-deploy-plugin</artifactId>
+        <configuration>
+          <skip>true</skip>
```

Comment:
Seems we were `deploy`ing an empty JAR on every core release, which is pointless. (The `src/main/java/` was long since moved to `jenkins-test-harness`.)